dashboard/test/ui/features/curriculum_platform/levelbuilder/lesson_edit_page.feature

```diff
@@ -28,3 +28,38 @@ Feature: Using the Lesson Edit Page
     And element ".uitest-activity-duration-input" has value "15"
 
     And I delete the temp script and lesson
+
+  Scenario: Add a level using the lesson edit page
+    Given I create a levelbuilder named "Levi"
+    And I create a temp script and lesson
+    And I view the temp lesson edit page
+    And I wait until element ".uitest-activity-card" is visible
+    And element ".uitest-open-add-level-button" is visible
+    And element ".uitest-bubble" contains text "1"
+    And element ".uitest-bubble" does not contain text "2"
+
+    # Open the Add Level dialog, search for an artist level and add the first one
+
+    # TODO: use regular click instead once button responds to click events
+    When execute JavaScript expression "$('.uitest-open-add-level-button').simulate('mousedown')"
+    And I wait until element "h2" contains text "Add Levels"
+    And element "#add-level-type" is visible
+    And I select the "Artist" option in dropdown "add-level-type"
+    And element ".fa-search" is visible
+    And I press ".fa-search" using jQuery
+    And I wait until element "td" contains text "Artist"
+    And element ".fa-plus" is visible
+    And I press the first ".fa-plus" element
+    # TODO: change to not use jquery once button is not obscured
```

Comment:
![Screen Shot 2020-10-22 at 1 08 51 PM](https://user-images.githubusercontent.com/8001765/96924647-cf7a0700-1467-11eb-88a1-8ce9bfb9fca0.png)

Reply:
for context, jquery is able to click elements which are not obscured whereas the selenium actions intentionally fail if you try to interact with an element which is obscured since that indicates that the user might not be able to click it either.

Reply:
I have a PR to fix this: https://github.com/code-dot-org/code-dot-org/pull/37421

Reply:
OK, that one looks good to go so feel free to merge it and then I'll update this to click without jquery.